mne/preprocessing/ica.py

```diff
@@ -1367,7 +1377,8 @@ def _check_exclude(self, exclude):
         if exclude is None:
             return list(set(self.exclude))
         else:
-            return list(set(self.exclude + exclude))
+            return list(set(self.exclude).union(set(exclude)))
+            # Allow both self.exclude and exclude to be array-like.
```

Comment:
It would be better if the comment preceded the code line it refers to, currenlty it is the last line in a method which is a bit weird. :)

Reply:
ok